in-progress/8754-slashing-and staking.md

```diff
@@ -0,0 +1,357 @@
+
+The prover's bond sits in escrow (or a separate contract) for $t=4$ weeks. During this period, the prover can plead their case and convince the validators to vote to unslash them. A validator must initiate a vote to unslash the prover and submit to L1 for verification on time, otherwise the bond gets slashed. 
+
+Since "Ethereum was congested" is a subjective statement, two ways we could implement this:
+
+1) Make $t$ larger, giving the prover enough time to convince the validators to manually update their software to vote to unslash the prover. This could be done on the forums or validator community channels for example. 
+2) Add an environment variable `max_eth_base_fee` that clients agree to implement. If `base_gas_fee > max_gas_fee` at the proof submission deadline time, then validator nodes automatically sign votes to unslash prover.
+
+They are largely equivalent except that the second requires less coordination.
+
+### PoG Non-slashable offences
+
+We would like to slash the following actions but this would require vast changes to the p2p and/or would still be susceptible to attacks. Still including them here for brevity but these offenses are NOT slashable in the implementation.  
+
+### 1. Sequencers don't accept EpochProofQuotes
+
+If no EpochProofQuote is accepted within $C=13$ slots and the epoch reorgs, validators check for whether sequencers could have accepted an actionable EpochProofQuote during their turn. 
+
+An actionable quote is one where:
+i) The timestamp of receiving it was before the $C=13$th slot of the epoch. 
+ii) The epoch information is correct. 
+iii) The prover had the required bond in escrow. 
+iv) The `basisPointFee` was within the allowable range. 
+
+Validators check for whether they saw such an actionable `EpochProofQuote`. If they did, they initiate a proposal to slash all committee members that could've accepted the quote but did not. 
+
+Why is this hard to implement?
+- We're designing S&P coordination to be out-of-protocol. The above assumes provers send quotes via the p2p.
+- Provers currently can send quotes even without having the necessary bond. 
+- Provers could send the quotes to a partition of the network, excluding the proposers who can activate the quote. This leads to timing games.
+
+### 2. Inactivity Leak
+
+Upon entering Based Fallback mode, validators can slash the committee members who have not submitted blocks during their slots. Since entering Based Fallback mode requires $T_{\textsf{fallback, enter}}$ time of no activity on the Pending Chain, validators initiate a proposal to slash the inactive committees during that time period. 
+
+Why is this hard to implement? 
+- A malicious committee could abuse the Resp/Req module to send attestations to a partition of the network but not the current proposers. While the validators will then gossip the attestations back to the proposer, they may not make it back on time. Again timing games. 
```

Comment:
I don't follow the attack here.

Reply:
Presently, Req/Resp is only used for requesting TxObjects that a validator is missing, unless I'm mistaken @Maddiaa0 .

Reply:
correct, the protocol describes a message format that honest nodes will follow, they cannot be sent data that does not follow it. And it does not allow for accepting attestations. 

When amin asked about this I misunderstood what he was asking, which is why this is present here.